sos/report/plugins/networking.py

```diff
@@ -295,23 +294,19 @@ def setup(self):
         if self.policy.dist_version() in ubuntu_ss_kmods:
             self.ss_kmods = ubuntu_ss_kmods[self.policy.dist_version()]
 
-        super(UbuntuNetworking, self).setup()
+        super().setup()
 
         self.add_copy_spec([
-            "/etc/resolvconf",
+            "/etc/netplan/*.yaml",
             "/etc/network/interfaces",
             "/etc/network/interfaces.d",
             "/etc/resolv.conf",
-            "/run/netplan/*.yaml",
-            "/etc/netplan/*.yaml",
+            "/etc/resolvconf",
             "/lib/netplan/*.yaml",
+            "/run/netplan/*.yaml",
             "/run/systemd/network"
         ])
 
-        if self.get_option("traceroute"):
```

Comment:
Already collected in the super class. This class just sets a different `trace_host`.